test/functional/core/startup_spec.lua

```diff
@@ -40,6 +40,19 @@ describe('startup', function()
     ok(string.find(alter_slashes(meths.get_option_value('runtimepath', {})), funcs.stdpath('config'), 1, true) == nil)
   end)
 
+  it('prevents infinite loop', function()
+    clear({ '-u', 'NONE' })
+    local screen
+    screen = Screen.new(60, 3)
+    screen:attach()
+    command('terminal ' .. nvim_prog .. ' -u NONE --server $NVIM --remote-ui')
```

Comment:
This doesn't work with powershell or cmd.exe. I think this should work:
```suggestion
    funcs.termopen({ nvim_prog, '-u', 'NONE', '--server', meths.get_vvar('servername'), '--remote-ui' })
```